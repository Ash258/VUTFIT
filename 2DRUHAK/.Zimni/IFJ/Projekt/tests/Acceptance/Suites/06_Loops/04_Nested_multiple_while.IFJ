' Nested and mutliple while loop

scope
	dim a as double
	dim b as double
	a = 0 - 123.789
	do while a > (0 - 236.741)
		b = 9
		do while b > 3.02
			print 63;
			b = b - 1.123
		loop
		a = a + a * 2
	loop
	a = 78
	do while a > 55
		print 57;
		a = a - 5.38
	loop
end scope
