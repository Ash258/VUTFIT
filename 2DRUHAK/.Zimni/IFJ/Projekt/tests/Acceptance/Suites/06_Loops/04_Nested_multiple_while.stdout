 63 63 63 63 63 63 57 57 57 57 57